src/App.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-import App from "App";
 import { render, act } from "test_utils";
+import App from "App";
```

Comment:
Would pnp prevent this? 
The eslint rules that organize sorts explicitly sort internal deps after external ones and this breaks that. Is there a way we can do this without breaking that sort order?

Reply:
I _think_ making this change to the ESLint config would prevent all the import sort changes

https://github.com/evergreen-ci/spruce/pull/1010/files#diff-b3d81df455893664810b193106f3543c920c80f9f4281b5541b949a8e1790769

Reply:
I actually think this change breaks the minimatch expression matching used by `import/order`.
https://www.digitalocean.com/community/tools/glob?comments=true&glob=%7Banalytics%2Ccomponents%2Cconstants%2Ccontext%2Cgql%2Choc%2Chooks%2Cpages%2Ctest_utils%2Ctypes%2Cutils%7D%7B%2C%2F%2A%2A%7D%22&matches=false&tests=components%2Ftest&tests=analytics%2Ftest

https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2B%28analytics%7Ccomponents%7Cconstants%7Ccontext%7Cgql%7Choc%7Chooks%7Cpages%7Ctypes%7Cutils%29%2F%2A%2A&matches=false&tests=components%2Ftest&tests=analytics%2Ftest

Reply:
Hm, isn't the problem with your first link that it contains an extraneous double quote?
<img width="775" alt="image" src="https://user-images.githubusercontent.com/9298431/167178738-85383e2a-af8b-4d4a-a789-7e5b8cc2c9fe.png">

Removing it passes the checks:
https://www.digitalocean.com/community/tools/glob?comments=true&glob=%7Banalytics%2Ccomponents%2Cconstants%2Ccontext%2Cgql%2Choc%2Chooks%2Cpages%2Ctest_utils%2Ctypes%2Cutils%7D%7B%2C%2F%2A%2A%7D&matches=false&tests=components%2Ftest&tests=analytics%2Ftest

Reply:
Wow great catch! My copy and pasting skills aren't what they used to be.
It did not error in my editor earlier so i assumed the problem was with the statement. But i guess eslint needed to restart. 
![image](https://user-images.githubusercontent.com/4605522/167189937-5534f424-9b9e-4f18-9c94-4921dfaa3508.png)